examples/conversion-gen/Makefile

```diff
@@ -0,0 +1,16 @@
+TOOL=conversion-gen
```

Comment:
blargh, when did this repo grow makefiles :( :(

Reply:
long long ago